eyewear_shop/demo/stock_warehouse_orderpoint_replenish.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <function name="write" model="stock.warehouse.orderpoint">
+        <value eval="[ref('orderpoint_1')]"/>
+        <value eval="{'qty_to_order': 20.0}"/>
+    </function>
+    <function name="write" model="stock.warehouse.orderpoint">
+        <value eval="[ref('orderpoint_2')]"/>
+        <value eval="{'qty_to_order': 13.0}"/>
+    </function>
```

Comment:
Can't you add the `qty_to_order` in the definition of the record, in `stock_carehouse_orderpoint.xml`?

Reply:
Where did the "action_replenish" go?